odkx-src/app-designer-prereqs.rst

```diff
@@ -143,32 +143,41 @@ To install the Android SDK:
 
   .. note::
 
-    You can alternatively install the full Android Studio if you so wish, in which case you should follow Google's instructions and then skip to step 6 of this guide.
+    You can alternatively install the full Android Studio if you so wish, in which case you should follow Google's instructions and .
```

Comment:
there seems to be an "and ." that is confusing.

Reply:
I will fix that, didn't catch it before. Thank you.

Reply:
This reads more clearly now, just going to request that someone more familiar with the development side review!